client/my-sites/media-library/list-item-video.jsx

```diff
@@ -33,13 +34,10 @@ module.exports = React.createClass( {
 	},
 
 	render: function() {
-		var thumbnail = this.getHighestQualityThumbnail(),
-			url;
+		const thumbnail = this.getHighestQualityThumbnail();
 
 		if ( thumbnail ) {
-			// All thumbnails extracted from the media should be accessible via
```

Comment:
Should we leave this comment? It's the only place I saw documented that videos always use Photon, so it may be worth it to keep it.